model_compression_toolkit/core/common/quantization/quantization_params_generation/symmetric_selection.py

```diff
@@ -52,8 +52,14 @@ def symmetric_selection_tensor(tensor_data: np.ndarray,
         Optimal threshold to quantize the tensor in a symmetric manner.
     """
 
-    unsigned_tensor_data = np.abs(tensor_data)
```

Comment:
Why not use the unsigned tensor? (it is debatable what is better for symmetric quantization but it's not necessarily better to take the signed one so I just want to understand the reason)